### ISSUE TITLE
CSS class directive not recognised as used

### ISSUE THREAD

**Issue by palasjir** [tomblachut/svelte-intellij#115]
When using [class directives](https://svelte.dev/docs#class_name) used css class within style is marked as unused.

**Example:**
```javascript
<style>
.active { } // this is marked as unused by the editor
</style>

<div class:active></div>
```

**Expected**:
```javascript
<style>
.hello {} // this class is used
.active { } // this class is used
.disabled { } // this class is truly unused
</style>

<div class:active class="hello"></div>
```

**Comment by tomblachut** [comment]
Related to #11

**Comment by tomblachut** [comment]
Will be released as part of 0.16.0
